deploy/examples/import-external-cluster.sh

```diff
@@ -152,6 +171,56 @@ function importRGWAdminOpsUser() {
   --from-literal=secretKey="$RGW_ADMIN_OPS_USER_SECRET_KEY"
 }
 
+function createRBDStorageClass() {
+  provisioner=$OPERATOR_NAMESPACE+"rbd.csi.ceph.com"
+cat << eof | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: $RBD_STORAGE_CLASS_NAME
+provisioner: $provisioner
+parameters:
+  clusterID: $OPERATOR_NAMESPACE
```

Comment:
clusterID is namespace where ceph cluster is created not the operator namespace

Reply:
https://github.com/rook/rook/blob/f6a4e824f35df2252b9549b285f9d83c75d68ad9/deploy/examples/csi/cephfs/storageclass.yaml#L8,
I see this comment that why I have updated like this.

Reply:
https://github.com/rook/rook/blob/f6a4e824f35df2252b9549b285f9d83c75d68ad9/deploy/examples/csi/cephfs/storageclass.yaml#L8-L9
I think it meant the ceph cluster is running. that is why will see `# namespace:cluster` for the secret.